CONTRIBUTING.md

```diff
@@ -14,21 +14,16 @@ that benefit the community at large, not just code contributions:
 * Fix bugs and add new features
 
 EVE governance is conducted by the Technical Steering Committee (TSC),
-which is currently composed of 22 active members:
+which is currently composed of 10 active members:
 
```

Comment:
@GregChase I think changing 22 to 15 (sorry for forgetting to update it) is the only part of this patch that is still applicable. All the other individuals reached out personally to me and asked to still consider them to be part of the TSC. We can consider coming up with an "activity" criteria later on -- but for now there's no changes to this list needed.

Makes sense?

If it does -- can you please update the patch to just have 22 -> 15 change?